### ISSUE TITLE
StringExp is not returning the correct expression

### ISSUE THREAD

**Issue by realstraw** [krux/hyperion#216]
```scala
scala> StringExp("abc")
res0: com.krux.hyperion.expression.StringExp = #{abc}
```

It should be
```scala
scala> StringExp("abc")
res0: com.krux.hyperion.expression.StringExp = #{"abc"}
```

This should be fixed with https://github.com/krux/hyperion/issues/215 as `DateTimeFunctions` is depending on this bug.

**Comment by sethyates** [comment]
Is this really a bug?  If I read the docco correctly, it expects strings to be quoted: http://docs.aws.amazon.com/datapipeline/latest/DeveloperGuide/dp-pipeline-reference-functions-string.html

**Comment by sethyates** [comment]
Never mind.  I read that oppositely.  Carry on ;-)

**Comment by realstraw** [comment]
fixed by #233
